bluepysnap/circuit_validation.py

```diff
@@ -308,48 +313,81 @@ def _check_rotations():
     if missing_fields:
         errors.append(fatal('Group {} of {} misses biophysical fields: {}'.
                             format(group_name, group.file.filename, missing_fields)))
-    if 'components' not in config:
-        errors.append(fatal('No "components" in config'))
-        return errors
-    components = config['components']
-    errors += _check_components_dir('morphologies_dir', components)
-    errors += _check_components_dir('biophysical_neuron_models_dir', components)
+    _check_rotations()
```

Comment:
Why bring it back here? I moved it on purpose in the last commit to `master` branch.

Reply:
Oh, I must've messed it up in the merge. I'll review this.